Userland/Libraries/LibSoftGPU/Device.cpp

```diff
@@ -142,12 +138,6 @@ void Device::setup_blend_factors()
         VERIFY_NOT_REACHED();
     }
 
-    m_alpha_blend_factors.dst_constant = { 0.0f, 0.0f, 0.0f, 0.0f };
-    m_alpha_blend_factors.dst_factor_src_alpha = 0;
-    m_alpha_blend_factors.dst_factor_dst_alpha = 0;
-    m_alpha_blend_factors.dst_factor_src_color = 0;
-    m_alpha_blend_factors.dst_factor_dst_color = 0;
```

Comment:
Did you mean to say `= {}` here too?

Reply:
No. This is what is confusing. The default initialization is split in 2 places. Notice the first section is initializing the `src_*` members, then those are updated accordingly. Later, the second section is for `dst_*` members. This changes it to initialize all members then go about changes.